pom.xml

```diff
@@ -46,21 +46,37 @@
 
     <dependencyManagement>
         <dependencies>
+
+			<!-- Import dependency management start Spring Boot (because we don't use Spring-Boot as parent pom)-->
             <dependency>
-                <!-- Import dependency management from Spring Boot (because we don't use Spring-Boot as parent pom)-->
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-dependencies</artifactId>
                 <version>${spring-boot.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+
+			<!-- we should check if this dependency needs to be updated now and then-->
             <dependency>
                 <groupId>org.springframework.cloud</groupId>
                 <artifactId>spring-cloud-dependencies</artifactId>
                 <version>Finchley.RC2</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+
+			<dependency>
+				<groupId>dev.morphia.morphia</groupId>
+				<artifactId>core</artifactId>
+				<version>${morphia.version}</version>
```

Comment:
Just to be sure; this is a regular version isn't it? Not a 'beta/development' version? I'm asking because of the dev.morphia package name.

Reply:
That's right. The package names were changed around that version. The beta version is at 2.0 at the time I updated it for Fulltext.